rumqttd/src/router/routing.rs

```diff
@@ -193,7 +193,10 @@ impl Router {
         }
 
         // A connection should not be scheduled multiple times
-        debug_assert!(self.scheduler.check_readyqueue_duplicates());
+        #[cfg(debug_assertions)]
+        if !self.scheduler.check_readyqueue_duplicates() {
+            warn!("Connection was scheduled multiple times in readyqueue, this shouldn't happen.");
```

Comment:
Better to log the connection_id's that are duplicate in readyqueue as well so that we can check logs of those connection ids then and there and try to debug

Reply:
changed it to print `readyqueue` when duplicates where found, instead of just the duplicates which might be helpful